go/vt/sqlparser/ast.go

```diff
@@ -8202,6 +8209,13 @@ func (d InjectedExpr) walkSubtree(visit Visit) error {
 	return nil
 }
 
+// OrderedInjectedExpr is an InjectedExpr that has an OrderBy clause associated with it
+type OrderedInjectedExpr struct {
+	InjectedExpr
+	OrderBy OrderBy
+	Name    ColIdent
```

Comment:
Is `Name` used anywhere? I looked, but couldn't find a reference.

Reply:
Nope, failed experiment. I'll delete.